### PR TITLE
Feature/red test

### PR DESCRIPTION
**Pull request by Jancis**
- Initial redis subchart integration
- Chart name validation (will fail if contains one of enabled subcharts like "mailhog", "memcache" or "redis"). This is a special quirk for resource naming in bitnami subcharts https://github.com/bitnami/charts/blob/master/bitnami/common/templates/_names.tpl#L26

### CODE REVIEW

**Review by guncha25 — APPROVED** [review]
LGTM

File: charts/drupal/values.yaml [hunk 1]
```diff
@@ -575,6 +575,22 @@ memcached:
     # MaxItemSize
     - -I 4M
 
+# https://github.com/bitnami/charts/blob/master/bitnami/redis/values.yaml
+redis:
+  architecture: standalone
```
Comment (Rade333) [inline charts/drupal/values.yaml:580]: Should we add the `enabled: false` line here?

Comment (Jancis) [reply charts/drupal/values.yaml:581]: Good idea, fixed!

Comment (Jancis) [reply charts/drupal/values.yaml:597]: and then i set it to `enabled: true` :facepalm:

Comment (Jancis) [reply charts/drupal/values.yaml:597]: false alarm, it was true only in chart tests

**Pull request merged**